tasty-script-exitcode/src/Test/Tasty/HUnit/ScriptExit.hs

```diff
@@ -109,8 +118,16 @@ withPython3venv requirements todo =
                    ":\nstdout: " <> pipStdout <> "\nstderr: " <> pipStderr
                  (ExitSuccess, _, _) ->
                    pure ()
-         in do traverse (\reqPath -> pip ["install", "-r", reqPath]) requirements
-               todo pip venvPython
+         in do (exitCode, stdout, stderr) <- readCreateProcessWithExitCode pipInstall ""
+               case exitCode of
+                 ExitFailure code ->
+                   assertFailure $
+                   "Failed to create install `pip` with code " <>
```

Comment:
I'm not sure what "create install" means here, and I would be quite confused with this test failure.

Reply:
Oops! That definitely doesn't make sense. :) Fixed!